src/wp-includes/rest-api/endpoints/class-wp-rest-templates-controller.php

```diff
@@ -227,10 +227,11 @@ public function get_item_permissions_check( $request ) {
 	 * @return WP_REST_Response|WP_Error
 	 */
 	public function get_item( $request ) {
+		$id = str_replace( '%20', ' ', $request['id'] );
```

Comment:
This should be moved to `_sanitize_template_id` callback.

Reply:
Also, should we use url decode functions here.

Reply:
Yup, you're right. Thanks @spacedmonkey. Done in https://github.com/WordPress/wordpress-develop/pull/2034/commits/03bf811e413e6e76d4a864dce0cda339dd930346.